docs/Options/GenerateOptions.md

```diff
@@ -1,8 +1,10 @@
 # Generate Options
 Options are optional and case-insensitive
 
+* `--overwrite` 
+    * Overwrite existing output files. Options are: `true` or `false` (default)
 * `--violate` 
-    * generate data which violates profile constraints. Options are: `true` or `false` (default)
+    * generate data which violates profile constraints. Options are: `true` or `false` (default). When this switch is used the output target must be an empty directory.
```

Comment:
"...must be an empty directory" only when _overwrite_ is also specified?

Reply:
that statement has been removed